include/swift/AST/DiagnosticsSema.def

```diff
@@ -4188,8 +4188,12 @@ ERROR(async_call_without_await_in_autoclosure,none,
 ERROR(async_call_without_await_in_async_let,none,
       "call is 'async' in an 'async let' initializer that is not marked "
       "with 'await'", ())
+ERROR(async_prop_access_without_await,none,
+      "property access is 'async' but is not marked with 'await'", ())
+ERROR(async_subscript_access_without_await,none,
+      "subscript access is 'async' but is not marked with 'await'", ())
 WARNING(no_async_in_await,none,
-        "no calls to 'async' functions occur within 'await' expression", ())
+        "no 'async' operations occur within 'await' expression", ())
```

Comment:
I guess because we don't say we "call" a property? operations reads weird... is it expressions?

Reply:
I wanted to say "expressions", but then we would have the word "expression" in the sentence twice, so I went with the generic combination of "call" and "access" as "operation". 🤷🏼